### PR TITLE
Migrate errorablefileinput

### PR DESCRIPTION
**Pull request by phiden**

### CODE REVIEW

File: src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  errorMessage: PropTypes.string,
+
+  /* string */
+  buttonText: 'Add Files',
+
+  /* file types */
+  mimeTypes: '',
+
+  /* accepts multiple files */
+  multiple: false
```
Comment (rianfowler) [inline src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx]: Remove `buttonText`, `mimeTypes`, `multiple` from the `propTypes` object.

Comment (phiden) [reply src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx]: so if we do that, it breaks the template renderer -- see the conversation in the design system channel -- but would it work to remove the default props instead?

File: src/components/form/controls/ErrorableFileInput/ErrorableFileInput.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    };
+
+    tree.getMountedInstance().handleChange(event);
+
+    expect(onChange.called).to.be.true;
+    expect(event.target.value).to.be.null;
+  });
+
+  it('passes aXe check', () => {
+    const check = axeCheck(<ErrorableFileInput buttonText="my label" onChange={onChange}/>);
```
Comment (rianfowler) [inline src/components/form/controls/ErrorableFileInput/ErrorableFileInput.unit.spec.jsx]: Probably need to remove the `onChange` prop here.

File: src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx (hunk 1 above)
Comment (rianfowler) [reply src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx]: Oh I see now. 

We can remove `buttonText: 'Add Files,` line- `buttonText` has a `propType` defined for it on line 84. 

In the `propTypes` object, update these two properties: 
`mimeTypes: '',` => `mimeTypes: PropTypes.string,`
`multiple: false` => `multiple: PropTypes.bool`. 

The `defaultProps` object should work correctly the way it is.

Comment (phiden) [reply src/components/form/controls/ErrorableFileInput/ErrorableFileInput.jsx]: done, thanks for the direction!

**Comment by phiden** [comment]
@rianfowler any idea why Jenkins doesn't like the path to tooltip? It works fine locally.

**Pull request merged**